### PR TITLE
Added WorkItemTrackingProcess and WorkItemTrackingProcessDefinitions APIs

### PR DESCRIPTION
**Pull request by ojferro**
Exposed these APIs, as they are also available on docs.microsoft.com:

Added WorkItemTrackingProcessApi, WorkItemTrackingProcessDefinitionsApi, WorkItemTrackingProcessInterfaces and WorkItemTrackingProcessDefinitionsInterfaces, and made corresponding changes to WebApi.

### CODE REVIEW

**Comment by stephenmichaelf** [comment]
@ojferro Could you provide some context for these changes? What are they? How did they come about? Are these changes going into the other libraries too?

Thanks!

File: api/WebApi.ts [hunk 1]
```diff
@@ -423,4 +424,19 @@ export class WebApi {
             return decryptedContent;
         }
     }
+
+    public async getWorkItemTrackingProcessDefinitionApi(serverUrl?: string, handlers?: VsoBaseInterfaces.IRequestHandler[]): Promise<workitemtrackingprocessdefinitionm.IWorkItemTrackingApi> {
+        // TODO: Load RESOURCE_AREA_ID correctly.
+        serverUrl = await this._getResourceAreaUrl(serverUrl || this.serverUrl, "5264459e-e5e0-4bd8-b118-0985e68a4ec5");
```
Comment (stephenmichaelf) [inline api/WebApi.ts]: Same resource area id for both? Just want to make sure.

**Comment by stephenmichaelf** [comment]
Have these been generated for the other SDKs too? Python, C#, etc.

Please add a sample in [samples\creation.ts](https://github.com/Microsoft/vsts-node-api/blob/master/samples/creation.ts)

If you could email me stfrance at microsoft.com with the steps you used to generate the APIs that would be helpful so I can make sure they are updated in the future.

File: api/WebApi.ts (hunk 1 above)
Comment (ojferro) [reply api/WebApi.ts]: Yes, WorkItemTracking, WorkItemTrackingProcess, and WorkItemTrackingProcessDefinitions all have the same area ID

**Comment by stephenmichaelf** [comment]
For visibility, coordinating with @ojferro over email on a few things.

**Comment by stephenmichaelf** [comment]
Can you add:

`printSectionStart('Dashboard');`

With the name of each section? It will keep things consistent and easy to follow the output.

Are there any other changes you wanted to do? I will do a final review tomorrow but I think it looks good.

**Comment by ojferro** [comment]
@stephenmichaelf  Added `printSectionStart('...')`
No more changes to be made for the node API, working on generating APIs for other SDKs.

**Pull request merged**